### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by dleehr**
- Moves django project up one directory to repo root
- Includes Dockerfile to build web application image
- Changes to support postgres instead of sqlite (Includes rebuilding migrations)
- Includes docker-compose file to run this app in a container with data stored in a postgres container

Launching docker-compose requires a handoverservice.env file with the following values:

    HANDOVERSERVICE_SECRET_KEY=<Randomly-generated Django secret key>
    HANDOVERSERVICE_DDSCLIENT_URL=<API endpoint for DukeDS>
    HANDOVERSERVICE_DDSCLIENT_AGENT_KEY=<DukeDS Agent Key>

### CODE REVIEW

**Comment by dleehr** [comment]
Fixes #12

**Comment by dleehr** [comment]
@johnbradley You're welcome to review this if you like. It's actually pretty short - most of the changes are moving files around, making a copy of the settings, and re-generating migrations for postgres compatibility

File: README.md [hunk 1]
```diff
@@ -24,7 +24,6 @@ Installation
 
 5. Create the database schema:
 
-        $ cd handoverservice
         $ python manage.py migrate
 
 6. Start the app:
```
Comment (johnbradley) [inline README.md]: You may want to consider adding instructions for docker if that is the preferred method of running this in production.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+version: '2'
+services:
+  db:
+    image: postgres
```
Comment (johnbradley) [inline docker-compose.yml]: Do you need to specify the postgres docker environment variables?
I think it would work if they were already in your shell.

**Comment by johnbradley** [comment]
LGTM.

File: docker-compose.yml (hunk 2 above)
Comment (dleehr) [inline docker-compose.yml]: Good catch. I left the defaults from the postgres image and they are hard-coded into the settings.docker file

File: README.md (hunk 1 above)
Comment (dleehr) [inline README.md]: Fixed in 3d4af5a56b09ba7ac5e5c26af57b0fd425b4b577

File: docker-compose.yml (hunk 2 above)
Comment (dleehr) [inline docker-compose.yml]: Fixed in 9afe11b11243c1708e5eb501b2859e9277f7c814

File: docker-compose.yml [hunk 3]
```diff
@@ -11,3 +11,7 @@ services:
       - "8000:8000"
     depends_on:
       - db
+    entrypoint:
```
Comment (dleehr) [inline docker-compose.yml]: @johnbradley This was a ugly-but-not-too-ugly way to use the wait-for-postgres script without making it the default entrypoint. I added the script to the source repo (and therefore the docker image), but it's only activated under docker-compose. So the docker image by itself isn't bound to use it.

The original docker image does have one dependency on the compose configuration - it expects the database to be on a host called `db`. I see you extracted that out in your app, and I suppose I should too.

**Pull request merged**